plugins/mcreator-core/features/height_provider_uniform.json

```diff
@@ -0,0 +1,28 @@
+{
+  "args0": [
+    {
+      "type": "input_value",
+      "name": "min",
+      "check": "VerticalAnchor"
+    },
+    {
+      "type": "input_value",
+      "name": "max",
+      "check": "VerticalAnchor"
+    }
+  ],
+  "inputsInline": true,
+  "output": "HeightProvider",
+  "colour": 180,
+  "mcreator": {
+    "toolbox_id": "heightproviders",
+    "toolbox_init": [
+      "<value name=\"min\"><block type=\"vertical_anchor_above_bottom\"><field name=\"value\">0</field></block></value>",
+      "<value name=\"max\"><block type=\"vertical_anchor_below_top\"><field name=\"value\">0</field></block></value>"
+    ],
+    "inputs": [
+      "min",
+	  "max"

```

Comment:
You should rather fix input arrays indentations.

Reply:
Good catch